samples/snippets/create_table_schema_from_json.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pathlib
+
+
+def create_table(table_id: str) -> None:
+    orig_table_id = table_id
+    current_directory = pathlib.Path(__file__).parent
+    orig_schema_path = str(current_directory / "schema.json")
+    # [START bigquery_schema_file_create]
+    from google.cloud import bigquery
+
+    client = bigquery.Client()
+
+    # current_directory = pathlib.Path(__file__).parent
```

Comment:
Remove this line from the code sample. We don't want folks reading this code sample on cloud.google.com to see commented-out code.

```suggestion
```